docs.md

```diff
@@ -25,7 +25,7 @@ We are actively working on additional tutorials. Thanks for your patience.
 
 ### appleseed Apps, Plugins and Tools
 
-- [appleseed for Blender](https://appleseed-blenderseed.readthedocs.io/en/latest/)
+- [appleseed for Blender](https://appleseed-blenderseed.readthedocs.io)
```

Comment:
Ah, then we should do the same on the website and `README.md` on GitHub.

I can take care of the website.

Reply:
The readme.md is fine.  The link goes to the root page of the readthedocs site and I use the controls there to determine which version of the manual is loaded by default.

Reply:
Actually... https://appleseed-blenderseed.readthedocs.io/en/latest/ is different from https://appleseed-blenderseed.readthedocs.io/en/master/. Are you sure we should change the link?

Reply:
Yep.  Take a look at the blenderseed README.  It links to https://appleseed-blenderseed.readthedocs.io/ When you click on the link it sends you to the 2.0.0 docs (to match the official version).

readthedocs has a control where you can set what version that link pulls up.

![image](https://user-images.githubusercontent.com/13521172/69434382-1f55c880-0d03-11ea-9cb1-0f76fc9a894d.png)

Reply:
Ok. My only question then is how do you link to subpages, i.e. what's the default version of this URL: https://appleseed-maya.readthedocs.io/en/master/shaders/shaders.html

Reply:
That's a full link so it'll go exactly where it says it will (the 'master' version of the Maya docs).

The 'default version' control only comes into play when you don't specify a specific doc build in your URL.

Reply:
I understand, but can I create a link to the shaders page of the default version? My link above specifically goes to the master version, which isn't what I'd like.

Reply:
I don't think so.  I think the default only works when you use https://appleseed-blenderseed.readthedocs.io/

Reply:
Ok, thanks!